File: TextosJapones/src/main/java/org/jisho/textosJapones/util/converter/FloatConverter.java
package org.jisho.textosJapones.util.converter;

import javafx.util.StringConverter;

public class FloatConverter extends StringConverter<Float> {

	@Override
	public String toString(Float object) {
		if (object == null)
			return null;
		return String.valueOf(object);
	}

	@Override
	public Float fromString(String string) {
		if (string == null || string.isEmpty())
			return null;
		return Float.valueOf(string);
	}

}

File: ProcessaTexto/src/main/kotlin/br/com/fenix/processatexto/util/converter/FloatConverter.kt
package br.com.fenix.processatexto.util.converter

import javafx.util.StringConverter


class FloatConverter : StringConverter<Float?>() {

    override fun toString(float: Float?): String? {
        return float?.toString()
    }

    override fun fromString(string: String?): Float? {
        return if (string == null || string.isEmpty()) null else string.toFloat()
    }
}